src/ufoLib2/objects/layer.py

```diff
@@ -148,7 +148,7 @@ def unlazify(self) -> None:
 
     __deepcopy__ = _deepcopy_unlazify_attrs
 
-    def __contains__(self, name: str) -> bool:
+    def __contains__(self, name: object) -> bool:
```

Comment:
Why object and not str?

Reply:
Ah, right. It's what in `Mapping`'s API.

Reply:
Uh. Alright then